### ISSUE TITLE
Investigate content migration from development to production

### ISSUE THREAD

**Issue by jgmorse** [mlibrary/heliotrope#463]
It appears ideal that the batch importer would never have to be run in production; instead, we'd run it in development, then migrate that content to production.  This would have the following benefits:
- Lowers impact of troubleshooting the batch ingest process (with new content or metadata needs) that either break the current process or necessitate the wiping out of ingested content and starting over.
- Eliminates impact on production performance for CPU-intensive jobs.
- Allows Handles to be registered and fixed before the content is live.

Check with the community to see if there is a practice around doing this, known pitfalls to avoid, etc.
- [ ] IDs assigned in development must remain consistent after migration to production.
- [ ] Handle URLs and/or DOIs should be registered and assigned to objects (and added to the metadata) while in development; post-migration metadata should be generated to facilitate updating handles/DOIs to resolve to the new URL.  (This is probably its own issue.)
- [ ] Import/export by heliotrope namespace/URL
- [ ] Import/export specific content (by press, or by list of IDs) -- this must include all auto-generated objects associated with those IDs.

Refer to #556.

**Comment by jgmorse** [comment]
This looks very promising, but will need some testing:
https://github.com/fcrepo4-labs/fcrepo-import-export

Use cases here (we can/should add if necessary):
https://wiki.duraspace.org/display/FF/Design+-+Import+-+Export

**Comment by jgmorse** [comment]
Let's use the Michigan title (Animal Acts) to test this, since there are fewer dependencies on the exact timing of the deployment (no external party to coordinate with).

**Comment by sethaj** [comment]
Blocked by #556

**Comment by gkostin1966** [comment]
I'm going to use this card to learn all about Fedora.
